examples/keepalive/README.md

```diff
@@ -0,0 +1,16 @@
+# Keepalive
+
+This example illustrates how to set up client-side keepalive pings and
+server-side keepalive pings and connection age and idleness settings.
```

Comment:
readability nit: "...set up client and server keepalive pings as well as connection..."

Reply:
Honestly, I'm not sure. I'm trying to convey that the keepalive ping settings are on both the client and the server, but the connection age and idleness settings are only on the server. The Go example uses this wording:

> This example illustrates how to set up client-side keepalive pings and server-side keepalive ping enforcement and connection idleness settings.

Which is similar, but also omits the fact that it has server-side keepalive pings.

Reply:
Oh I see, on initial read this distinction did not come through, but makes sense now that you mentioned it. If you wanted to be super clear you go with:

"...how to set up keepalive pings on the client-side and keepalive pings, connection age and idleness setting on the server-side"

But I'll leave it up to you.